packages/@aws-cdk/cfn-include/lib/cdk-include.ts

```diff
@@ -0,0 +1,221 @@
+import * as core from '@aws-cdk/core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+/**
+ * Construction properties of {@link CdkInclude}.
+ */
+export interface CdkIncludeProps {
+  /**
+   * Path to the template file.
+   *
+   * Currently, only JSON templates are supported.
+   */
+  readonly templateFile: string;
+
+  /**
+   * Whether the resources should have the same logical IDs in the resulting CDK template
+   * as they did in the original CloudFormation template file.
+   *
```

Comment:
Mention that `getResource` still accepts the original IDs

Reply:
That's more about the contract of `getResource`, and it's already documented there. Doesn't feel super necessary to repeat it here.

Reply:
I felt it might be appropriate to indicate this here given this controls the IDs... no need to be skimpy about docs, right?

Reply:
Sure. Done.